src/scripts/modules/media/body/body.less

```diff
@@ -79,6 +79,17 @@ body {
   .transition(background 0.5s ease-in-out);
 }
 
+@media (max-width: @screen-xs-max) {
```

Comment:
You can try even simpler trick because it's already using flexbox so you will not have to use `@media`.
Just set `flex-wrap: wrap;` to `.media-body-about-description-wrapper` and `min-width: 400px;` to `.media-body-about-description`. It will automaticall go below cover when hit `min-height`.

You can experiment with `flex: 1;` on `<img>` and move `margin-bottom` from `...wrapper` to `...cover` and so on.

Reply:
I'm okay with that